docs/fleet/fragments.md

```diff
@@ -300,6 +300,9 @@ Here are the version options:
 {{% /tab %}}
 {{< /tabs >}}
 
+After configuring fragment modifications, check your machine's [**LOGS** tab](/cloud/machines/#logs).
+If there are problems with fragment modifications, depending on the issue, your machine or some of its components may not work until the configuration is fixed.
```

Comment:
I would say "may not work as expected" instead of "may not work". This is because if the machine was previously working with the old config, even if the overwrite fails, it will continue to work. It won't be the new config with the overwrite applied though.

Reply:
I overwrote a pin value with a negative value for a motor I think. The overwrite succeeded but the resulting change causes an illegal config for the component. So it in facet may not continue to work. I suppose you can argue that the overwrite succeeded but you still get logged errors and the component is no longer appearing on the control tab if I do that

Reply:
Ah I see. yeah the overwrite succeeded in that case because the field existed; we can't tell if the pin value itself is wrong - that happens on the robot